### ISSUE TITLE
Improved Chargebar

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#1314]
this fits better than just a circle imo but to each their own, just sharing as a food for thought as i always say
https://steamcommunity.com/sharedfiles/filedetails/?id=2276828271

**Comment by TiredFoks** [comment]
I don't really mind the circles, but this also looks nice, so I wouldn't mind it aswell if it was a thing.

**Comment by Lonrite** [comment]
> I don't really mind the circles, but this also looks nice, so I wouldn't mind it aswell if it was a thing.

yeah I like the circle, i just think it looks better with those subtle black dots around. (obviously without this red background lol)
